### PR TITLE
Mac OSX support

### PR DESCRIPTION
**Pull request by pearu**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pearu** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,48 +11,46 @@ source:
...
-    - llvmdev 7.*   # [osx], see above
-    - clangdev 7.*  # [osx], see above
-    - openssl
+    - blosc
+    - libgdal 2.*   # with libgdal-3 one ImportTest fails
+    - arrow-cpp >=0.12.1
     - thrift-cpp >=0.11.0
     - ncurses
-    # omnscidb server calls clang++ in runtime for UDF support
-    - {{ compiler('cxx') }}   # [linux] required for C++ headers (when using clang)
```
Comment (xmnlab) [inline recipe/meta.yaml:51]: `fatal error: 'cassert' file not found`

not sure if ir could be failing because `compiler(cxx)` was removed.

**Comment by xmnlab** [comment]
osx building worked! :clap:

File: recipe/meta.yaml (hunk 1 above)
Comment (pearu) [reply recipe/meta.yaml:51]: Fixed.

**Comment by pearu** [comment]
@xmnlab , please review and merge if acceptable.

Meanwhile I'll create another PR for cleaning up the build scripts and fixing any issues from actually deploying omniscidb-cpu.

**Review by xmnlab — APPROVED** [review]
Looks great @pearu ! Great job!

**Pull request merged**